src/main/java/mate/project/validation/FieldMatch.java

```diff
@@ -0,0 +1,20 @@
+package mate.project.validation;
+
+import jakarta.validation.Constraint;
+import jakarta.validation.Payload;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Constraint(validatedBy = FieldMatchValidator.class)
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface FieldMatch {
+    String message() default "Fields do not match";
+    Class<?>[] groups() default {};
+    Class<? extends Payload>[] payload() default {};
+
+    String first();
```

Comment:
There is no documentation or comments indicating what 'first()' and 'second()' methods are supposed to represent. It might be helpful to provide documentation for these methods to indicate their purpose.